lib/users.ts

```diff
@@ -0,0 +1,35 @@
+import dbConnector from "@lib/integration/dbConnector";
+import executeQuery from "@lib/integration/queryManager";
+import { AuthenticatedUser } from "@lib/types";
+import { logMessage } from "./logger";
+
+/**
+ * Get all users
+ * @returns An array of all Users
+ */
+export const getUsers = async (): Promise<AuthenticatedUser[]> => {
+  const result = await executeQuery(
```

Comment:
Should there be a try/catch around this block of code?

Reply:
Yes there should.  Thanks for catching.  I hadn't done a complete recheck of everything yet in the PR, hence the draft PR status.